Example/Teller.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BuildSystemType</key>
+	<string>Original</string>
```

Comment:
This change happened because CocoaPods is not working with XCode 10 at the moment. This is the workaround. 

https://github.com/CocoaPods/CocoaPods/issues/8073